.golangci.yml

```diff
@@ -0,0 +1,171 @@
+---
+run:
+  # concurrency: 6
+  timeout: 2m
+  issues-exit-code: 1
+  tests: true
+linters-settings:
+  goheader:
+    template-path: ".license/template.txt"
+    values:
+      const:
+        year: 2020
+      regexp:
+        year-range: ((\d\d\d\d-{{year}})|({{year}}))
+        company: .*
+        copyright-holder: Copyright \(c\) {{year-range}} {{company}}\n\n
+        copyright-holders: ({{copyright-holder}})+
+  errcheck:
+    check-type-assertions: false
+    check-blank: false
+  govet:
+    check-shadowing: true
+    settings:
+      printf:
+        funcs:
+          - (github.com/sirupsen/logrus.FieldLogger).Infof
+          - (github.com/sirupsen/logrus.FieldLogger).Warnf
+          - (github.com/sirupsen/logrus.FieldLogger).Errorf
+          - (github.com/sirupsen/logrus.FieldLogger).Fatalf
+  golint:
+    min-confidence: 0.8
+  goimports:
+    local-prefixes: github.com/networkservicemesh/sdk
```

Comment:
So... I'm pretty sure this is the wrong local-prefix...

Reply:
Nice catch! Will fix.